packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -150,6 +162,13 @@ export const CreateAlertDefinition = () => {
             serviceType={serviceTypeWatcher}
           />
           <CloudPulseAlertSeveritySelect name="severity" />
+          <MetricCriteriaField
+            getMaxInterval={(interval: number) =>
+              setMaxScrapeInterval(interval)
+            }
```

Comment:
rename the prop name getMaxInterval, because it is actually not getting anything instead it is setting max scrape interval

Reply:
Makes sense, changed it to setMaxInterval